README.md

```diff
@@ -128,17 +128,18 @@ Or add the path to the `bin` folder to your `$PATH` and invoke `swiftgen` direct
 
 The tool is provided as a unique `swiftgen` binary command-line, with the following subcommands:
 
-* `swiftgen assets [OPTIONS] DIR`
-* `swiftgen colors [OPTIONS] FILE`
-* `swiftgen fonts [OPTIONS] DIR`
-* `swiftgen storyboards [OPTIONS] DIR`
-* `swiftgen strings [OPTIONS] FILE`
+* `swiftgen colors [OPTIONS] FILE1 …`
+* `swiftgen fonts [OPTIONS] DIR1 …`
+* `swiftgen storyboards [OPTIONS] DIR1 …`
+* `swiftgen strings [OPTIONS] FILE1 …`
+* `swiftgen xcassets [OPTIONS] FILE1 …`
```

Comment:
For Finder, it's a folder, for xcode, it's a file (maybe package).

Crazy idea, but what about `CATALOG1`? (or `CAT1`)